embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/EmbraceNetworkConnectivityServiceTest.kt

```diff
@@ -96,85 +95,6 @@ internal class EmbraceNetworkConnectivityServiceTest {
         verify { context.unregisterReceiver(service) }
     }
 
-    @Test
```

Comment:
Do we have equivalent tests for the data source or are these functionality implemented by the infra the data source builds on?

Reply:
So, the new data source doesn't care about the actual network status or connection type. I can write a new set of unit tests to verify that each type is returned properly by `EmbraceNetworkConnectivityService` but I don't see much value there TBH.